src/lottie/lottieproxymodel.h

```diff
@@ -334,4 +340,22 @@ class LOTProxyModel<LOTFillData>
     LOTFilter                  mFilter;
 };
 
+template <>
+class LOTProxyModel<LOTGroupData>
+{
+public:
+    LOTProxyModel(LOTGroupData *model): _modelData(model) {}
+    LOTFilter& filter() {return mFilter;}
+    const char* name() const {return _modelData->name();}
+    VPointF point(int frame) const
```

Comment:
point is a good name? how about position?

Reply:
I will change to position